src/entities/mentor/ui/mentor-feedback-card/mentor-feedback-card.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+
+import { useState } from 'react';
+import classNames from 'classnames/bind';
+import Image from 'next/image';
+import { MentorFeedback } from '../../types';
+import { Modal } from '@/shared/ui/modal';
+
+import styles from './mentor-feedback-card.module.scss';
+
+const cx = classNames.bind(styles);
+
+type MentorFeedbackCardProps = MentorFeedback;
+
+const FEEDBACK_MAX_CHARS = 500;
+
+export const MentorFeedbackCard = ({ name, course, review, photo }: MentorFeedbackCardProps) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const handleOpenModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const handleCloseModal = () => {
+    setIsModalOpen(false);
+  };
+
+  const isLongReview = review.length > FEEDBACK_MAX_CHARS;
+
+  const card = (
+    <div className={cx('card')}>
+      <div className={cx('card-info')} data-testid="card-info">
+        <Image
+          className={cx('card-picture')}
+          src={photo}
+          alt={`${name} ${course}`}
+          data-testid="mentor-photo"
+        />
+        <header className={cx('card-header')}>
+          <h3 className={cx('card-title')} data-testid="card-title">
```

Comment:
Can we use our Subtitle component here instead `h3` and `h4`?

Reply:
I replaced only `h3` with `Subtitle`, since our component renders only the h3 tag. Maybe we should create a story about adding a new prop and then refactor it in all the places

Reply:
Will you create a new task for this?

Reply:
https://github.com/rolling-scopes/site/issues/742